lib/basedriver/capabilities.js

```diff
@@ -53,19 +53,30 @@ function validateCaps (caps, constraints = {}, skipPresenceConstraint = false) {
 }
 
 // Standard, non-prefixed capabilities (see https://www.w3.org/TR/webdriver/#dfn-table-of-standard-capabilities)
-const STANDARD_CAPS = ['browserName', 'browserVersion', 'platformName', 'acceptInsecureCerts', 'pageLoadStrategy', 'proxy', 'setWindowRect', 'timeouts', 'unhandledPromptBehavior'];
+const STANDARD_CAPS = [
+  'browserName',
+  'browserVersion',
+  'platformName',
+  'acceptInsecureCerts',
+  'pageLoadStrategy',
+  'proxy',
+  'setWindowRect',
+  'timeouts',
+  'unhandledPromptBehavior'
+];
 
 function isStandardCap (cap) {
-  return !!_.find(STANDARD_CAPS, (standardCap) => standardCap.toLowerCase() === cap.toLowerCase());
+  return !!_.find(STANDARD_CAPS, (standardCap) => standardCap.toLowerCase() === `${cap}`.toLowerCase());
 }
 
 // If the 'appium:' prefix was provided and it's a valid capability, strip out the prefix (see https://www.w3.org/TR/webdriver/#dfn-extension-capabilities)
+// (NOTE: Method is destructive and mutates contents of caps)
 function stripAppiumPrefixes (caps) {
   const prefix = 'appium:';
   const prefixedCaps = _.filter(_.keys(caps), cap => cap.startsWith(prefix));
   const badPrefixedCaps = [];
   const unprefixedCaps = _.filter(_.keys(caps), (cap) => (
-    cap.indexOf(':') < 0 && !isStandardCap(cap)
+    cap.includes(':') && !isStandardCap(`${cap}`)
```

Comment:
it is not necessary to do it here if conversation to string is already done in `isStandardCap`